packages/backend/src/backend.ts

```diff
@@ -0,0 +1,7 @@
+export interface IBackend{
```

Comment:
The interface cannot be empty, please add the required functions.

Reply:
added functions in interfaces